tests/stripe-emulator.test.ts

```
@@ -0,0 +1,218 @@
+import type Stripe from 'stripe'
+import { randomUUID } from 'node:crypto'
+import { createServer } from 'node:net'
+import { afterAll, afterEach, beforeAll, describe, expect, it, vi } from 'vitest'
+import { createEmulator } from 'emulate'
```

ESLint

[error] Expected "emulate" to come before "vitest".

(perfectionist/sort-imports)